web.test.tests/web.test.tests/Tests/SettingsPageTests.cs

```diff
@@ -41,34 +41,28 @@ public void AfterEachTest()
         [Test]
         public void CancelBtn_Closes_Settings()
         {
-            SettingsPage settingsPage = new SettingsPage(driver);
-            settingsPage.CancelBtn.Click();
+            new SettingsPage(driver).CancelBtn.Click();
 
             new WebDriverWait(driver, TimeSpan.FromMilliseconds(10000)).Until(ExpectedConditions.VisibilityOfAllElementsLocatedBy(By.Id("amount")));
```

Comment:
I suggest to move this to method on SettingsPage:
```
public DepositePage Cancel()
{
    CancelBtn.Click();
    new WebDriverWait... 
    return new DepositePage(driver);
}
```

Reply:
Done